[31Mar2023 15:01:44.051] [main/INFO] [cpw.mods.modlauncher.Launcher/MODLAUNCHER]: ModLauncher running: args [--launchTarget, forgeclientuserdev, --version, MOD_DEV, --assetIndex, 1.19, --assetsDir, C:\Users\User\.mcreator\gradle\caches\forge_gradle\assets, --gameDir, ., --fml.forgeVersion, 43.2.0, --fml.mcVersion, 1.19.2, --fml.forgeGroup, net.minecraftforge, --fml.mcpVersion, 20220805.130853]
[31Mar2023 15:01:44.056] [main/INFO] [cpw.mods.modlauncher.Launcher/MODLAUNCHER]: ModLauncher 10.0.8+10.0.8+main.0ef7e830 starting: java version 17.0.5 by Eclipse Adoptium; OS Windows 10 arch amd64 version 10.0
[31Mar2023 15:01:44.205] [main/INFO] [mixin/]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=union:/C:/Users/User/.mcreator/gradle/caches/modules-2/files-2.1/org.spongepowered/mixin/0.8.5/9d1c0c3a304ae6697ecd477218fa61b850bf57fc/mixin-0.8.5.jar%23120!/ Service=ModLauncher Env=CLIENT
[31Mar2023 15:01:44.689] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModFileParser/LOADING]: Mod file C:\Users\User\.mcreator\gradle\caches\modules-2\files-2.1\net.minecraftforge\javafmllanguage\1.19.2-43.2.0\5713dfd49576886a7ffc353f14f067ad42369592\javafmllanguage-1.19.2-43.2.0.jar is missing mods.toml file
[31Mar2023 15:01:44.694] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModFileParser/LOADING]: Mod file C:\Users\User\.mcreator\gradle\caches\modules-2\files-2.1\net.minecraftforge\lowcodelanguage\1.19.2-43.2.0\51c88985dd125457fb8b6c2c58a4f3b9349a6b57\lowcodelanguage-1.19.2-43.2.0.jar is missing mods.toml file
[31Mar2023 15:01:44.700] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModFileParser/LOADING]: Mod file C:\Users\User\.mcreator\gradle\caches\modules-2\files-2.1\net.minecraftforge\mclanguage\1.19.2-43.2.0\f4fa83ef8129c7f7cc5847fbafb07f7ae86c1bb\mclanguage-1.19.2-43.2.0.jar is missing mods.toml file
[31Mar2023 15:01:44.706] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModFileParser/LOADING]: Mod file C:\Users\User\.mcreator\gradle\caches\modules-2\files-2.1\net.minecraftforge\fmlcore\1.19.2-43.2.0\d4dcb5b809b7f035ac22b058941cf81f84451545\fmlcore-1.19.2-43.2.0.jar is missing mods.toml file
[31Mar2023 15:01:44.793] [main/INFO] [net.minecraftforge.fml.loading.moddiscovery.JarInJarDependencyLocator/]: No dependencies to load found. Skipping!
[31Mar2023 15:01:46.293] [main/INFO] [cpw.mods.modlauncher.LaunchServiceHandler/MODLAUNCHER]: Launching target 'forgeclientuserdev' with arguments [--version, MOD_DEV, --gameDir, ., --assetsDir, C:\Users\User\.mcreator\gradle\caches\forge_gradle\assets, --assetIndex, 1.19]
[31Mar2023 15:01:50.087] [pool-3-thread-1/INFO] [net.minecraft.util.datafix.DataFixers/]: Building unoptimized datafixer
[31Mar2023 15:01:51.822] [Render thread/WARN] [net.minecraft.server.packs.VanillaPackResources/]: Assets URL 'union:/C:/Users/User/.mcreator/gradle/caches/forge_gradle/minecraft_user_repo/net/minecraftforge/forge/1.19.2-43.2.0_mapped_official_1.19.2/forge-1.19.2-43.2.0_mapped_official_1.19.2-recomp.jar%23183!/assets/.mcassetsroot' uses unexpected schema
[31Mar2023 15:01:51.823] [Render thread/WARN] [net.minecraft.server.packs.VanillaPackResources/]: Assets URL 'union:/C:/Users/User/.mcreator/gradle/caches/forge_gradle/minecraft_user_repo/net/minecraftforge/forge/1.19.2-43.2.0_mapped_official_1.19.2/forge-1.19.2-43.2.0_mapped_official_1.19.2-recomp.jar%23183!/data/.mcassetsroot' uses unexpected schema
[31Mar2023 15:01:51.834] [Render thread/INFO] [com.mojang.authlib.yggdrasil.YggdrasilAuthenticationService/]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[31Mar2023 15:01:51.838] [Render thread/INFO] [net.minecraft.client.Minecraft/]: Setting user: Dev
[31Mar2023 15:01:51.942] [Render thread/INFO] [net.minecraft.client.Minecraft/]: Backend library: LWJGL version 3.3.1 build 7
[31Mar2023 15:01:55.971] [modloading-worker-0/INFO] [net.minecraftforge.common.ForgeMod/FORGEMOD]: Forge mod loading, version 43.2.0, for MC 1.19.2 with MCP 20220805.130853
[31Mar2023 15:01:55.972] [modloading-worker-0/INFO] [net.minecraftforge.common.MinecraftForge/FORGE]: MinecraftForge v43.2.0 Initialized
[31Mar2023 15:01:56.999] [Render thread/INFO] [net.minecraftforge.gametest.ForgeGameTestHooks/]: Enabled Gametest Namespaces: []
[31Mar2023 15:01:57.132] [Render thread/INFO] [com.mojang.text2speech.NarratorWindows/]: Narrator library for x64 successfully loaded
[31Mar2023 15:01:57.158] [Render thread/INFO] [net.minecraft.server.packs.resources.ReloadableResourceManager/]: Reloading ResourceManager: Default, Mod Resources
[31Mar2023 15:01:57.910] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [forge] Starting version check at https://files.minecraftforge.net/net/minecraftforge/forge/promotions_slim.json
[31Mar2023 15:01:58.962] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [forge] Found status: UP_TO_DATE Current: 43.2.0 Target: null
[31Mar2023 15:02:03.086] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:block/uran_ore' referenced from: exacraft:uran_ore#: com.google.gson.JsonParseException: exacraft:blocks/уран_ore is not valid resource location
[31Mar2023 15:02:03.088] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:block/uran_block' referenced from: exacraft:uran_block#: com.google.gson.JsonParseException: exacraft:blocks/уран_ore_block is not valid resource location
[31Mar2023 15:02:03.089] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:block/torii_ore' referenced from: exacraft:torii_ore#: com.google.gson.JsonParseException: exacraft:blocks/торий_ore is not valid resource location
[31Mar2023 15:02:03.090] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:block/torii_block' referenced from: exacraft:torii_block#: com.google.gson.JsonParseException: exacraft:blocks/торий_ore_block is not valid resource location
[31Mar2023 15:02:03.092] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:block/rubin_ore' referenced from: exacraft:rubin_ore#: com.google.gson.JsonParseException: exacraft:blocks/рубин_ore is not valid resource location
[31Mar2023 15:02:03.092] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:block/rubin_block' referenced from: exacraft:rubin_block#: com.google.gson.JsonParseException: exacraft:blocks/рубин_ore_block is not valid resource location
[31Mar2023 15:02:03.093] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:block/aliuminii_ore' referenced from: exacraft:aliuminii_ore#: com.google.gson.JsonParseException: exacraft:blocks/алюминий_ore is not valid resource location
[31Mar2023 15:02:03.094] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:block/aliuminii_block' referenced from: exacraft:aliuminii_block#: com.google.gson.JsonParseException: exacraft:blocks/алюминий_ore_block is not valid resource location
[31Mar2023 15:02:03.096] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:block/sviniets_ore' referenced from: exacraft:sviniets_ore#: com.google.gson.JsonParseException: exacraft:blocks/свинец_ore is not valid resource location
[31Mar2023 15:02:03.097] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:block/sviniets_block' referenced from: exacraft:sviniets_block#: com.google.gson.JsonParseException: exacraft:blocks/свинец_ore_block is not valid resource location
[31Mar2023 15:02:03.102] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:block/titan_ore' referenced from: exacraft:titan_ore#: com.google.gson.JsonParseException: exacraft:blocks/титан_ore is not valid resource location
[31Mar2023 15:02:03.104] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:block/titan_block' referenced from: exacraft:titan_block#: com.google.gson.JsonParseException: exacraft:blocks/титан_ore_block is not valid resource location
[31Mar2023 15:02:03.105] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:block/eliektronit_ore' referenced from: exacraft:eliektronit_ore#: com.google.gson.JsonParseException: exacraft:blocks/электронит_ore is not valid resource location
[31Mar2023 15:02:03.106] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:block/eliektronit_block' referenced from: exacraft:eliektronit_block#: com.google.gson.JsonParseException: exacraft:blocks/электронит_ore_block is not valid resource location
[31Mar2023 15:02:03.107] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:block/vardonit_ore' referenced from: exacraft:vardonit_ore#: com.google.gson.JsonParseException: exacraft:blocks/вардонит_ore is not valid resource location
[31Mar2023 15:02:03.109] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:block/vardonit_block' referenced from: exacraft:vardonit_block#: com.google.gson.JsonParseException: exacraft:blocks/вардонит_ore_block is not valid resource location
[31Mar2023 15:02:03.110] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:block/plastal_ore' referenced from: exacraft:plastal_ore#: com.google.gson.JsonParseException: exacraft:blocks/пласталь_ore is not valid resource location
[31Mar2023 15:02:03.112] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:block/plastal_block' referenced from: exacraft:plastal_block#: com.google.gson.JsonParseException: exacraft:blocks/пласталь_ore_block is not valid resource location
[31Mar2023 15:02:03.113] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:block/osmii_ore' referenced from: exacraft:osmii_ore#: com.google.gson.JsonParseException: exacraft:blocks/осмий_ore is not valid resource location
[31Mar2023 15:02:03.114] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:block/osmii_block' referenced from: exacraft:osmii_block#: com.google.gson.JsonParseException: exacraft:blocks/осмий_ore_block is not valid resource location
[31Mar2023 15:02:05.120] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:plastal_armor_helmet#inventory' referenced from: exacraft:plastal_armor_helmet#inventory: com.google.gson.JsonParseException: exacraft:items/пласталь_head is not valid resource location
[31Mar2023 15:02:05.121] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:plastal_armor_chestplate#inventory' referenced from: exacraft:plastal_armor_chestplate#inventory: com.google.gson.JsonParseException: exacraft:items/пласталь_body is not valid resource location
[31Mar2023 15:02:05.121] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:plastal_armor_leggings#inventory' referenced from: exacraft:plastal_armor_leggings#inventory: com.google.gson.JsonParseException: exacraft:items/пласталь_leggings is not valid resource location
[31Mar2023 15:02:05.122] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:plastal_armor_boots#inventory' referenced from: exacraft:plastal_armor_boots#inventory: com.google.gson.JsonParseException: exacraft:items/пласталь_boots is not valid resource location
[31Mar2023 15:02:05.123] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:vardonit_armor_helmet#inventory' referenced from: exacraft:vardonit_armor_helmet#inventory: com.google.gson.JsonParseException: exacraft:items/вардонит_head is not valid resource location
[31Mar2023 15:02:05.123] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:vardonit_armor_chestplate#inventory' referenced from: exacraft:vardonit_armor_chestplate#inventory: com.google.gson.JsonParseException: exacraft:items/вардонит_body is not valid resource location
[31Mar2023 15:02:05.124] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:vardonit_armor_leggings#inventory' referenced from: exacraft:vardonit_armor_leggings#inventory: com.google.gson.JsonParseException: exacraft:items/вардонит_leggings is not valid resource location
[31Mar2023 15:02:05.125] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:vardonit_armor_boots#inventory' referenced from: exacraft:vardonit_armor_boots#inventory: com.google.gson.JsonParseException: exacraft:items/вардонит_boots is not valid resource location
[31Mar2023 15:02:05.135] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:uran_ingot#inventory' referenced from: exacraft:uran_ingot#inventory: com.google.gson.JsonParseException: exacraft:items/уран_ingot is not valid resource location
[31Mar2023 15:02:05.136] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:torii#inventory' referenced from: exacraft:torii#inventory: com.google.gson.JsonParseException: exacraft:items/торий_gem is not valid resource location
[31Mar2023 15:02:05.136] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:rubin#inventory' referenced from: exacraft:rubin#inventory: com.google.gson.JsonParseException: exacraft:items/рубин_gem is not valid resource location
[31Mar2023 15:02:05.137] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:aliuminii_ingot#inventory' referenced from: exacraft:aliuminii_ingot#inventory: com.google.gson.JsonParseException: exacraft:items/алюминий_ingot is not valid resource location
[31Mar2023 15:02:05.138] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:osmii#inventory' referenced from: exacraft:osmii#inventory: com.google.gson.JsonParseException: exacraft:items/осмий_gem is not valid resource location
[31Mar2023 15:02:05.138] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:osmii_ingot#inventory' referenced from: exacraft:osmii_ingot#inventory: com.google.gson.JsonParseException: exacraft:items/осмий_ingot is not valid resource location
[31Mar2023 15:02:05.139] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:sviniets_ingot#inventory' referenced from: exacraft:sviniets_ingot#inventory: com.google.gson.JsonParseException: exacraft:items/свинец_ingot is not valid resource location
[31Mar2023 15:02:05.140] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:titan_ingot#inventory' referenced from: exacraft:titan_ingot#inventory: com.google.gson.JsonParseException: exacraft:items/титан_ingot is not valid resource location
[31Mar2023 15:02:05.141] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:eliektronit_dust#inventory' referenced from: exacraft:eliektronit_dust#inventory: com.google.gson.JsonParseException: exacraft:items/электронит_dust is not valid resource location
[31Mar2023 15:02:05.141] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:vardonit#inventory' referenced from: exacraft:vardonit#inventory: com.google.gson.JsonParseException: exacraft:items/вардонит_gem is not valid resource location
[31Mar2023 15:02:05.142] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:plastal_ingot#inventory' referenced from: exacraft:plastal_ingot#inventory: com.google.gson.JsonParseException: exacraft:items/пласталь_ingot is not valid resource location
[31Mar2023 15:02:05.143] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:vardonit_axe#inventory' referenced from: exacraft:vardonit_axe#inventory: com.google.gson.JsonParseException: exacraft:items/вардонит_axe is not valid resource location
[31Mar2023 15:02:05.144] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:vardonit_pickaxe#inventory' referenced from: exacraft:vardonit_pickaxe#inventory: com.google.gson.JsonParseException: exacraft:items/вардонит_pickaxe is not valid resource location
[31Mar2023 15:02:05.144] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:vardonit_sword#inventory' referenced from: exacraft:vardonit_sword#inventory: com.google.gson.JsonParseException: exacraft:items/вардонит_sword is not valid resource location
[31Mar2023 15:02:05.145] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:vardonit_shovel#inventory' referenced from: exacraft:vardonit_shovel#inventory: com.google.gson.JsonParseException: exacraft:items/вардонит_shovel is not valid resource location
[31Mar2023 15:02:05.145] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:vardonit_hoe#inventory' referenced from: exacraft:vardonit_hoe#inventory: com.google.gson.JsonParseException: exacraft:items/вардонит_hoe is not valid resource location
[31Mar2023 15:02:05.146] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:plastal_pickaxe#inventory' referenced from: exacraft:plastal_pickaxe#inventory: com.google.gson.JsonParseException: exacraft:items/пласталь_pickaxe is not valid resource location
[31Mar2023 15:02:05.147] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:plastal_axe#inventory' referenced from: exacraft:plastal_axe#inventory: com.google.gson.JsonParseException: exacraft:items/пласталь_axe is not valid resource location
[31Mar2023 15:02:05.147] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:plastal_sword#inventory' referenced from: exacraft:plastal_sword#inventory: com.google.gson.JsonParseException: exacraft:items/пласталь_sword is not valid resource location
[31Mar2023 15:02:05.148] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:plastal_shovel#inventory' referenced from: exacraft:plastal_shovel#inventory: com.google.gson.JsonParseException: exacraft:items/пласталь_shovel is not valid resource location
[31Mar2023 15:02:05.149] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to load model: 'exacraft:plastal_hoe#inventory' referenced from: exacraft:plastal_hoe#inventory: com.google.gson.JsonParseException: exacraft:items/пласталь_hoe is not valid resource location
[31Mar2023 15:02:05.509] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to resolve texture reference: #missingno in exacraft:item/osmii_ore
[31Mar2023 15:02:05.509] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to resolve texture reference: #missingno in exacraft:item/aliuminii_ore
[31Mar2023 15:02:05.509] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to resolve texture reference: #missingno in exacraft:item/eliektronit_ore
[31Mar2023 15:02:05.509] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to resolve texture reference: #missingno in exacraft:item/sviniets_block
[31Mar2023 15:02:05.509] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to resolve texture reference: #missingno in exacraft:item/uran_ore
[31Mar2023 15:02:05.509] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to resolve texture reference: #missingno in exacraft:item/plastal_ore
[31Mar2023 15:02:05.509] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to resolve texture reference: #missingno in exacraft:item/rubin_block
[31Mar2023 15:02:05.510] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to resolve texture reference: #missingno in exacraft:item/titan_block
[31Mar2023 15:02:05.510] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to resolve texture reference: #missingno in exacraft:item/torii_ore
[31Mar2023 15:02:05.510] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to resolve texture reference: #missingno in exacraft:item/eliektronit_block
[31Mar2023 15:02:05.510] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to resolve texture reference: #missingno in exacraft:item/uran_block
[31Mar2023 15:02:05.510] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to resolve texture reference: #missingno in exacraft:item/aliuminii_block
[31Mar2023 15:02:05.510] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to resolve texture reference: #missingno in exacraft:item/sviniets_ore
[31Mar2023 15:02:05.510] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to resolve texture reference: #missingno in exacraft:item/torii_block
[31Mar2023 15:02:05.510] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to resolve texture reference: #missingno in exacraft:item/titan_ore
[31Mar2023 15:02:05.510] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to resolve texture reference: #missingno in exacraft:item/vardonit_block
[31Mar2023 15:02:05.510] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to resolve texture reference: #missingno in exacraft:item/plastal_block
[31Mar2023 15:02:05.510] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to resolve texture reference: #missingno in exacraft:item/vardonit_ore
[31Mar2023 15:02:05.510] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to resolve texture reference: #missingno in exacraft:item/rubin_ore
[31Mar2023 15:02:05.510] [Worker-Main-3/WARN] [net.minecraft.client.resources.model.ModelBakery/]: Unable to resolve texture reference: #missingno in exacraft:item/osmii_block
[31Mar2023 15:02:05.704] [Worker-Main-7/ERROR] [net.minecraft.client.renderer.texture.TextureAtlas/]: Using missing texture, file exacraft:textures/items/overworld_ec_gem.png not found
[31Mar2023 15:02:05.995] [Worker-Main-5/ERROR] [net.minecraft.client.renderer.texture.TextureAtlas/]: Using missing texture, file exacraft:textures/blocks/overworld_ec_ore.png not found
[31Mar2023 15:02:06.147] [Worker-Main-2/ERROR] [net.minecraft.client.renderer.texture.TextureAtlas/]: Using missing texture, file exacraft:textures/blocks/overworld_ec_ore_block.png not found
[31Mar2023 15:02:08.165] [Render thread/WARN] [net.minecraft.client.sounds.SoundEngine/]: Missing sound for event: minecraft:item.goat_horn.play
[31Mar2023 15:02:08.166] [Render thread/WARN] [net.minecraft.client.sounds.SoundEngine/]: Missing sound for event: minecraft:entity.goat.screaming.horn_break
[31Mar2023 15:02:08.211] [Render thread/INFO] [com.mojang.blaze3d.audio.Library/]: OpenAL initialized on device OpenAL Soft on Динамики (USB PnP Audio Device)
[31Mar2023 15:02:08.211] [Render thread/INFO] [net.minecraft.client.sounds.SoundEngine/SOUNDS]: Sound engine started
[31Mar2023 15:02:08.402] [Render thread/INFO] [net.minecraft.client.renderer.texture.TextureAtlas/]: Created: 1024x512x2 minecraft:textures/atlas/blocks.png-atlas
[31Mar2023 15:02:08.408] [Render thread/INFO] [net.minecraft.client.renderer.texture.TextureAtlas/]: Created: 256x128x2 minecraft:textures/atlas/signs.png-atlas
[31Mar2023 15:02:08.409] [Render thread/INFO] [net.minecraft.client.renderer.texture.TextureAtlas/]: Created: 512x512x2 minecraft:textures/atlas/banner_patterns.png-atlas
[31Mar2023 15:02:08.409] [Render thread/INFO] [net.minecraft.client.renderer.texture.TextureAtlas/]: Created: 512x512x2 minecraft:textures/atlas/shield_patterns.png-atlas
[31Mar2023 15:02:08.411] [Render thread/INFO] [net.minecraft.client.renderer.texture.TextureAtlas/]: Created: 256x256x2 minecraft:textures/atlas/chest.png-atlas
[31Mar2023 15:02:08.411] [Render thread/INFO] [net.minecraft.client.renderer.texture.TextureAtlas/]: Created: 512x256x2 minecraft:textures/atlas/beds.png-atlas
[31Mar2023 15:02:08.412] [Render thread/INFO] [net.minecraft.client.renderer.texture.TextureAtlas/]: Created: 512x256x2 minecraft:textures/atlas/shulker_boxes.png-atlas
[31Mar2023 15:02:09.585] [Render thread/WARN] [net.minecraft.client.renderer.ShaderInstance/]: Shader rendertype_entity_translucent_emissive could not find sampler named Sampler2 in the specified shader program.
[31Mar2023 15:02:09.840] [Render thread/INFO] [net.minecraft.client.renderer.texture.TextureAtlas/]: Created: 256x256x0 minecraft:textures/atlas/particles.png-atlas
[31Mar2023 15:02:09.840] [Render thread/INFO] [net.minecraft.client.renderer.texture.TextureAtlas/]: Created: 256x256x0 minecraft:textures/atlas/paintings.png-atlas
[31Mar2023 15:02:09.840] [Render thread/INFO] [net.minecraft.client.renderer.texture.TextureAtlas/]: Created: 128x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[31Mar2023 15:02:10.868] [Realms Notification Availability checker #1/INFO] [com.mojang.realmsclient.client.RealmsClient/]: Could not authorize you against Realms server: Invalid session id
[31Mar2023 15:02:20.964] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Loaded 7 recipes
[31Mar2023 15:02:22.111] [Render thread/INFO] [net.minecraft.advancements.AdvancementList/]: Loaded 1179 advancements
[31Mar2023 15:02:30.467] [Render thread/INFO] [net.minecraft.world.item.crafting.RecipeManager/]: Loaded 7 recipes
[31Mar2023 15:02:31.399] [Render thread/INFO] [net.minecraft.advancements.AdvancementList/]: Loaded 1179 advancements
[31Mar2023 15:02:32.042] [Server thread/INFO] [net.minecraft.client.server.IntegratedServer/]: Starting integrated minecraft server version 1.19.2
[31Mar2023 15:02:32.042] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Generating keypair
[31Mar2023 15:02:32.412] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Preparing start region for dimension minecraft:overworld
[31Mar2023 15:02:36.267] [Render thread/INFO] [net.minecraft.server.level.progress.LoggerChunkProgressListener/]: Preparing spawn area: 0%
[31Mar2023 15:02:36.267] [Render thread/INFO] [net.minecraft.server.level.progress.LoggerChunkProgressListener/]: Preparing spawn area: 0%
[31Mar2023 15:02:36.267] [Render thread/INFO] [net.minecraft.server.level.progress.LoggerChunkProgressListener/]: Preparing spawn area: 0%
[31Mar2023 15:02:36.267] [Render thread/INFO] [net.minecraft.server.level.progress.LoggerChunkProgressListener/]: Preparing spawn area: 0%
[31Mar2023 15:02:36.268] [Render thread/INFO] [net.minecraft.server.level.progress.LoggerChunkProgressListener/]: Preparing spawn area: 0%
[31Mar2023 15:02:36.268] [Render thread/INFO] [net.minecraft.server.level.progress.LoggerChunkProgressListener/]: Preparing spawn area: 0%
[31Mar2023 15:02:36.268] [Render thread/INFO] [net.minecraft.server.level.progress.LoggerChunkProgressListener/]: Preparing spawn area: 0%
[31Mar2023 15:02:36.268] [Render thread/INFO] [net.minecraft.server.level.progress.LoggerChunkProgressListener/]: Preparing spawn area: 0%
[31Mar2023 15:02:36.460] [Render thread/INFO] [net.minecraft.server.level.progress.LoggerChunkProgressListener/]: Preparing spawn area: 0%
[31Mar2023 15:02:36.960] [Render thread/INFO] [net.minecraft.server.level.progress.LoggerChunkProgressListener/]: Preparing spawn area: 24%
[31Mar2023 15:02:37.427] [Render thread/INFO] [net.minecraft.server.level.progress.LoggerChunkProgressListener/]: Preparing spawn area: 83%
[31Mar2023 15:02:37.621] [Render thread/INFO] [net.minecraft.server.level.progress.LoggerChunkProgressListener/]: Time elapsed: 5198 ms
[31Mar2023 15:02:37.623] [Server thread/INFO] [net.minecraftforge.server.permission.PermissionAPI/]: Successfully initialized permission handler forge:default_handler
[31Mar2023 15:02:37.743] [Server thread/INFO] [net.minecraft.client.server.IntegratedServer/]: Changing view distance to 5, from 10
[31Mar2023 15:02:37.746] [Server thread/INFO] [net.minecraft.client.server.IntegratedServer/]: Changing simulation distance to 12, from 0
[31Mar2023 15:02:38.228] [Netty Local Client IO #0/INFO] [net.minecraftforge.network.NetworkHooks/]: Connected to a modded server.
[31Mar2023 15:02:38.270] [Server thread/INFO] [net.minecraftforge.common.AdvancementLoadFix/]: Using new advancement loading for net.minecraft.server.PlayerAdvancements@222aeaf6
[31Mar2023 15:02:38.290] [Server thread/INFO] [net.minecraft.server.players.PlayerList/]: Dev[local:E:e19ca3c8] logged in with entity id 232 at (-43.56116706760091, 66.0, 58.096904965716185)
[31Mar2023 15:02:38.354] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Dev joined the game
[31Mar2023 15:02:39.682] [Render thread/INFO] [net.minecraft.advancements.AdvancementList/]: Loaded 40 advancements
[31Mar2023 15:02:45.974] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Dev: Set own game mode to Creative Mode]
[31Mar2023 15:02:45.983] [Render thread/INFO] [net.minecraft.client.gui.components.ChatComponent/]: [System] [CHAT] Set own game mode to Creative Mode
[31Mar2023 15:04:29.879] [Render thread/INFO] [net.minecraft.advancements.AdvancementList/]: Loaded 40 advancements
[31Mar2023 15:04:36.029] [Render thread/INFO] [net.minecraft.advancements.AdvancementList/]: Loaded 41 advancements
[31Mar2023 15:05:18.784] [Render thread/INFO] [net.minecraft.advancements.AdvancementList/]: Loaded 48 advancements
[31Mar2023 15:05:58.987] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Dev: Set own game mode to Survival Mode]
[31Mar2023 15:05:58.988] [Render thread/INFO] [net.minecraft.client.gui.components.ChatComponent/]: [System] [CHAT] Set own game mode to Survival Mode
[31Mar2023 15:06:23.308] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Dev: Set own game mode to Creative Mode]
[31Mar2023 15:06:23.309] [Render thread/INFO] [net.minecraft.client.gui.components.ChatComponent/]: [System] [CHAT] Set own game mode to Creative Mode
[31Mar2023 15:06:40.172] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Dev: Set own game mode to Survival Mode]
[31Mar2023 15:06:40.173] [Render thread/INFO] [net.minecraft.client.gui.components.ChatComponent/]: [System] [CHAT] Set own game mode to Survival Mode
[31Mar2023 15:07:11.141] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: [Dev: Set own game mode to Creative Mode]
[31Mar2023 15:07:11.142] [Render thread/INFO] [net.minecraft.client.gui.components.ChatComponent/]: [System] [CHAT] Set own game mode to Creative Mode
[31Mar2023 15:07:12.294] [Render thread/INFO] [net.minecraft.client.Minecraft/]: Stopping!
[31Mar2023 15:07:12.333] [Server thread/INFO] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Dev lost connection: Disconnected
[31Mar2023 15:07:12.333] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Dev left the game
[31Mar2023 15:07:12.348] [Server thread/INFO] [net.minecraft.server.network.ServerGamePacketListenerImpl/]: Stopping singleplayer server as player logged out
[31Mar2023 15:07:12.375] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Stopping server
[31Mar2023 15:07:12.375] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving players
[31Mar2023 15:07:12.375] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving worlds
[31Mar2023 15:07:13.420] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving chunks for level 'ServerLevel[New World]'/minecraft:overworld
[31Mar2023 15:07:13.618] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving chunks for level 'ServerLevel[New World]'/minecraft:the_end
[31Mar2023 15:07:13.620] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving chunks for level 'ServerLevel[New World]'/minecraft:the_nether
[31Mar2023 15:07:13.646] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: ThreadedAnvilChunkStorage (New World): All chunks are saved
[31Mar2023 15:07:13.646] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: ThreadedAnvilChunkStorage (DIM1): All chunks are saved
[31Mar2023 15:07:13.646] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: ThreadedAnvilChunkStorage (DIM-1): All chunks are saved
[31Mar2023 15:07:13.647] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: ThreadedAnvilChunkStorage: All dimensions are saved
